content/blog/2024/openscapes-sbg-workshop/index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Openscapes host a Surface Biology and Geology workshop
+date: "2024-05-29"
+banner:
+  image: "featured.png"
+authors: ["Yuvi Panda", "Jenny Wong"]
+tags: [geoscience, education]
+categories: [impact, updates, partnerships]
+featured: false
+draft: false
+---
+
+![Cover slide from the SBG Workshop](featured.png "Surface Biology and Geology: VITALS Workshop")
+
+[Openscapes](https://nasa-openscapes.github.io/) is a value-based initiative that advocates for kinder, better science produced with open source community and program building co-developed by Julia Lowndes and Erin Robinson. 
+
+The initiative recently supported the [Surface Biology and Geology: VITALS Workshop](https://nasa.github.io/VITALS/) hosted by NASA [Land Processes Distributed Activate Archive Center (LP DAAC)](https://lpdaac.usgs.gov/) and NASA [Jet Propulsion Laboratory (JPL)](https://www.jpl.nasa.gov/).
+
+Hands-on exercises teaching learners how to manipluate data collected from the [ECOSTRESS](https://ecostress.jpl.nasa.gov/) and [EMIT](https://earth.jpl.nasa.gov/emit/) instruments onboard the International Space Station were delivered using the 2i2c OpenScapes Hub. [Jupyter notebooks](https://nasa.github.io/VITALS/python/01_Finding_Concurrent_Data.html) were used to demonstrate how open source tools together with cloud data and compute resources could effectively analyse the the Canopy Water Content and the Land Surface Temperature over the [Jack and Laura Dangermond Preserve](https://www.dangermondpreserve.org/), Santa Barbara, CA.
```

Comment:
```suggestion
Hands-on exercises teaching learners how to manipluate data collected from the [ECOSTRESS](https://ecostress.jpl.nasa.gov/) and [EMIT](https://earth.jpl.nasa.gov/emit/) instruments onboard the International Space Station were delivered using the 2i2c Openscapes Hub. [Jupyter notebooks](https://nasa.github.io/VITALS/python/01_Finding_Concurrent_Data.html) were used to demonstrate how open source tools together with cloud data and compute resources could effectively analyse the the Canopy Water Content and the Land Surface Temperature over the [Jack and Laura Dangermond Preserve](https://www.dangermondpreserve.org/), Santa Barbara, CA.
```

Reply:
Only the "O" in Openscapes is capitalized; maybe do a spell check to see if there are others I missed :)

Reply:
Done!